src/__tests__/CurrentLocation.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CurrentLocation from '../components/CurrentLocation';
+
+const location = {
+  date: 'May 2',
+  name: 'Richmond',
+  aqi: 99,
+  description: 'Great air quality',
+  color: 'green',
+};
+
+function setup() {
```

Comment:
I found this in the React Components example on the Redux testing page:

http://redux.js.org/docs/recipes/WritingTests.html#components